index.js

```diff
@@ -2,16 +2,17 @@ var crypto = require('crypto');
 
 function EncryptedField(Sequelize, key, opt) {
     if (!(this instanceof EncryptedField)) {
-        return new EncryptedField(Sequelize, key);
+        return new EncryptedField(Sequelize, key, opt);
     }
 
     var self = this;
     self.key = new Buffer(key, 'hex');
     self.Sequelize = Sequelize;
-
+    
     opt = opt || {};
     self._algorithm = opt.algorithm || 'aes-256-cbc';
     self._iv_length = opt.iv_length || 16;
+    self._db_encoding = opt.db_encoding || undefined;
```

Comment:
by specifying undefined to buffer, what is the default it would use? Might be nice to just go ahead and specify the implicit node default explicitly to avoid hunting docs for what is going on in the future.